source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -12,15 +12,19 @@ Metrics and Alerts
    :local:
    :depth: 2
 
-.. admonition:: Metrics Version 2 Deprecated
+.. admonition:: Metrics version 3
    :class: note
 
```

Comment:
I want to say something at the top of the page like "This page covers version 3 of the metrics, introduced in blah blah. For version 2 of the metrics, see bling." Maybe even outside of the admonition, since people often skip those.

Reply:
rearranged some words so there's a bit of an introduction